columbo/_interaction.py

```diff
@@ -395,12 +424,12 @@ def ask(self, answers: Answers, no_user_input: bool = False) -> str:
                 no_user_input=no_user_input,
             )
 
-            error = self.validate(answer, answers)
-            if not error:
+            result = self._validate_and_convert(answer, answers)
+            if result.valid:
```

Comment:
Similar update to the one [here](https://github.com/wayfair-incubator/columbo/pull/88/files#r573085581).